### DISCUSSION TITLE
Option to disable tree view in Nemo

### DISCUSSION THREAD

**Discussion by MikeSesketcharba** [linuxmint/discussions#564] [Ideas]
I love the list view but I don't like to accidentally click the down arrows to expand the tree view. My suggestion is to add a checkbox to disable tree view, and have the list view as is.

![image](https://github.com/user-attachments/assets/16805352-0ee2-4eed-a383-a2690ffc2863)

Thanks!

**Comment by Darker7** [comment]
Already possible, simply uncheck this checkbox:

<img width="796" height="632" alt="image" src="https://github.com/user-attachments/assets/516f789d-15a3-4bde-9b26-5df3f4b32f48" />

:Ü™
